src/renderers/shared/ReactDebugTool.js

```diff
@@ -68,7 +69,7 @@ var ReactDebugTool = ((null: any): typeof ReactDebugTool);
 if (__DEV__) {
   const hooks = [];
   const didHookThrowForEvent = {};
-  const ReactComponentTreeHook = require('react/lib/ReactComponentTreeHook');
+  const ReactComponentTreeHook: ComponentTreeHookDevType = (require('react/lib/ReactComponentTreeHook'): any);
```

Comment:
That's what I thought, I'm a bit confused as to why they should have been removed if we don't want these errors to show when mixing bundles?